TradeItIosEmsApi/TradeItAuthLinkResult.h

```diff
@@ -13,6 +13,7 @@
 
 @property (nullable) NSString *userId;
 @property (nullable) NSString *userToken;
+@property (nullable) NSString *broker;
```

Comment:
We return `null` for non oauth flows -- should I leave this on here or move it to the `TradeItOAuthLinkResult` that inherits from this class? I don't think there's any problem leaving it here but it might be confusing later.